cloud/wfm-rest-api/README.md

```diff
@@ -109,6 +109,16 @@ Where `:objectId` is an `id` field of the object.
 
 Where `:objectId` is an `id` field of the object.
 
+### Search for objects
+
+Retrieves a list of objects that satisfies a query.
```

Comment:
that matches user query. Satisfies suggest equality. We using `LIKE` (wildcard) matching